src/utils/gamepad.ts

```diff
@@ -761,40 +770,80 @@ export class gamepads {
     this.delAllListeners(this._btnsListeners);
   }
 
+  removeInputCallbacks(inputName: string) {
```

Comment:
where this should be used and how? It looks like it doesn't remove the listeners but just remove from the Map.